CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Global owners
+* @newrelic/coreint
```

Comment:
```suggestion
* @newrelic-forks/coreint 
```

Reply:
I had to add the team as admin to the repo for this file to be valid too.

Good catch!